### ISSUE TITLE
test_decode_csv fails when running locally

### ISSUE THREAD

**Issue by dlesomewhere** [aws/sagemaker-huggingface-inference-toolkit#39]
When running the tests locally I encounter a failure:

```
>       assert decoded_data == {"inputs": ["I love you", "I like you"]}
E       AssertionError: assert {'inputs': [{'context': 'My name is Philipp and I live in Nuremberg',\n             'question': 'where do i live?'},\n            {'context': 'Berlin is the capital of Germany',\n             'question': 'where is Berlin?'}]} == {'inputs': ['I love you', 'I like you']}
E         Differing items:
E         {'inputs': [{'context': 'My name is Philipp and I live in Nuremberg', 'question': 'where do i live?'}, {'context': 'Berlin is the capital of Germany', 'question': 'where is Berlin?'}]} != {'inputs': ['I love you', 'I like you']}
E         Full diff:
E           {
E         -  'inputs': ['I love you',
E         -             'I like you'],
E         +  'inputs': [{'context': 'My name is Philipp and I live in Nuremberg',
E         +              'question': 'where do i live?'},
E         +             {'context': 'Berlin is the capital of Germany',
E         +              'question': 'where is Berlin?'}],
E           }

...

[gw0] FAILED tests/unit/test_decoder_encoder.py
```

This appears to be caused by a typo entering the wrong input data for the second assertion in the test.  I have proposed a PR to fix here - https://github.com/aws/sagemaker-huggingface-inference-toolkit/pull/38.

**Comment by philschmid** [comment]
Solved in #38 and #42

**Comment by dlesomewhere** [comment]
Thanks @philschmid !
